.github/workflows/e2e_test_branch.yml

```diff
@@ -0,0 +1,50 @@
+name: AWS E2E on test branch
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: [Deploy test branch]
+    types:
+      - completed
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read  # This is required for actions/checkout
+
+jobs:
+  RunE2E:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Git clone
+        uses: actions/checkout@v4
+      - name: configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
```

Comment:
Let's use commit SHA to be sure (and in other workflows for this action too)

Reply:
Done!